### ISSUE TITLE
AMA

### ISSUE THREAD

**Issue by Jerry-Hong** [Jerry-Hong/profiles#1]

**Comment by serapath** [comment]
Yeah sweet. Finally another person who also managed to even open the `AMA` issue already.
Many others did, but some did not manage yet.

I hope you found yesterday ok. If you have some feedback how to improve that would be great.

---
I checked your projects and it's obvious you are very passionate about rxobservables.

The `"projects"`  field was/is meant as open source github repositories that people want to learn or work on during the codecamp - so i'm not totally sure if this is how everyone understood it :P 
Do you plan to work on one or many of those repos?

I gues many beginners but probably also many experienced people are not yet very familiar with the concept of observables and for those who are - many have different opinions about which kind of observable is `"The Best ™"`, but you seem to like microsofts `rx` projects.
Many people I know prefer node streams or some other less opinionated way that does not prescribe you  a certain verbose vocabulary around how you can combine your observables, but instead gives you a generic way of combining your observable with a custom vocabluary. In node streams, every "observable" has the generic `.pipe(...)` method and you `require(...)` the custom vocabulary (e.g. a "map stream" or a "filter stream" or a "concatAll stream" or reduce, or zip, ...or whatever)

----

Anyways...
The goal is
1. everyone makes new empty open source github repos and describes their idea about what to learn or work on during the codecamp in a `README.md` or in an issue and links to them in `"projects": [...]` (you can also use an existing github repo you want to work on)
2. hopefully everyone goes around to discover other people who participate in the codecamp and see what ideas they have to get inspired and to refine your own idea with the goal to collaborate
  * so you might find beginners who might help you with some small tasks for your project and you could mentor them doing it with some code review and the likes
  * or you find other people who want to do ideas similar to yours and you could share work - so you build something they could re-use in their project as a module using `require` or maybe as an `iframe` ...but also a "traditional API" is possible in case you do something more backend-related - like ...maybe learn and work on "smart contracts" for blockchains?


We hope people discover each other and come up with ideas how to build projects that are either useful for others in their projects  or collaborate in other ways **AND THEN** during the weekend, we check everyone's progress and try to supply custom learning materials based on what they decided they want to learn and/or build :-)

**Comment by serapath** [comment]
did you already make a github repository for your project idea and have a link?
